lib/modulesync.rb

```diff
@@ -177,4 +178,29 @@ def self.update(cli_options)
     end
     exit 1 if errors && options[:fail_on_warnings]
   end
+
+  def self.execute(cli_options)
+    @options = config_defaults.merge(cli_options)
+
+    managed_modules.each do |puppet_module|
+      $stdout.puts "#{puppet_module.given_name}:"
+
+      puppet_module.repository.clone unless puppet_module.repository.cloned?
+      puppet_module.repository.switch branch: @options[:branch]
+
+      command_args = cli_options[:command_args]
+      local_script = File.expand_path command_args[0]
+      command_args[0] = local_script if File.exist?(local_script)
+
+      FileUtils.chdir(puppet_module.working_directory) do
```

Comment:
What's the benefit of `FileUtils.chdir` over `Dir.chdir`?

Reply:
In our context, none.
I toggle this conversation as this is a commit on top of this which remove this line.